mods/default/craftitems.lua

```diff
@@ -79,12 +79,13 @@ local max_text_size = 10000
 local max_title_size = 80
 local short_title_size = 35
 minetest.register_on_player_receive_fields(function(player, formname, fields)
-	if formname ~= "default:book" then return end
```

Comment:
Such a filter is required to avoid side-effects when submitting formspecs which have the same or similar `fields`.

Reply:
Sorry, I will fix it immediatelly.